_posts/2024-02-06-openttd-truetype-fonts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TrueType fonts for OpenTTD
+author: Zephyris
+---
+
+How do you update a 30-year old pixel font to make it more suitable for today?
+The pixel-art OpenTTD fonts are iconic, but were designed for 640x480 CRT monitors rather than today's 4k monsters.
+Without the horizontal smoothing of CRT scanlines, modern pixel-perfect rendering and 2x or 4x upscaling of the old bitmap/sprite-based fonts makes the text jagged and hard to read.
+
+<!-- more -->
+
+![Composite of a simulated CRT and modern screenshot of OpenTTD]({% link /static/img/post_2024-02-06-openttd-truetype-fonts/crt.png %}){:width="100%"}
+
+The short answer is "make a real font" - a TrueType font.
+The smooth splines of TrueType font characters scale with no loss of details and no pixel jaggies, while anti-aliased rendering mimics CRT scanline smoothing.
+
+Making a font which captures the look and feel of a pixel art font is an interesting challenge.
+In my misspent youth I've made some fonts from scratch, but designing a font to match the look and feel of a pixel art font is much more constrained.
+In some ways this limits creativity, but designing with constraints can be better.
```

Comment:
There are two missing commas here:
```suggestion
In my misspent youth, I've made some fonts from scratch, but designing a font to match the look and feel of a pixel art font is much more constrained.
In some ways, this limits creativity, but designing with constraints can be better.
```

Reply:
I disagree with this, fwiw

Possibly "I've" -> "I"